internal/controllers/nmc_reconciler.go

```diff
@@ -993,8 +1014,32 @@ func (p *podManagerImpl) baseWorkerPod(
 			MountPath: "/usr/lib/modules",
 			ReadOnly:  true,
 		},
+		{
+			Name:      volNameTmp,
+			MountPath: sharedFilesDir,
+			ReadOnly:  true,
+		},
 	}
 
+	const initContainerArgTmpl = `
+mkdir -p {{.KmodsHostPath}};
```

Comment:
IMHO KmodHostPath is not a very good name. We are not coping to host filesystem, but to the worker container filesystem

Reply:
I agree in is not the best name. I will find something else.

Reply:
I have renamed all the struct fields to:
```
KmodsPathSrc   
KmodsPathDst   
FirmwarePathSrc
FirmwarePathDst
```

Reply:
please see my comment below